src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -236,13 +237,16 @@ export async function initialize({
         localInvokeCommand = new DefaultSamLocalInvokeCommand(channelLogger, [WAIT_FOR_DEBUGGER_MESSAGES.PYTHON])
     }
 
-    const invokeLambda = async (args: LambdaLocalInvokeParams & { runtime: string }) => {
+    const invokeLambda = async (
```

Comment:
Nit: Optional for this PR, but this should be converted to `async function invokeLambda(args: ...): Promise<...> { ... }`

Reply:
✔️